src/NuGet.Core/NuGet.Build.Tasks/NuGet.props

```diff
@@ -0,0 +1,28 @@
+﻿<!--
+***********************************************************************************************
+NuGet.props
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (C) Microsoft Corporation. All rights reserved.
+***********************************************************************************************
+-->
+
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!-- 
+        Import Directory.Packages.props.
+    -->
+  <PropertyGroup Condition="'$(DirectoryPackagesPropsPath)' == ''">
```

Comment:
Can you add a condition that can disable the scan entirely? Something analogous to `$(ImportDirectoryBuildProps)`. Probably `$(ImportDirectoryPackagesProps)`.

Reply:
Added